src/Implement/UnitTestRequestHandler.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <arpa/inet.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <functional>
+#include <thread>
+
+#include "RequestHandler.h"
+
+void TestClient() {
+  int sockfd = 0;
+  sockfd = socket(AF_INET, SOCK_STREAM, 0);
+  struct sockaddr_in info;
+  bzero(&info, sizeof(info));
+  info.sin_family = PF_INET;
+  info.sin_addr.s_addr = inet_addr("127.0.0.1");
+  info.sin_port = htons(9000);
+  int err = connect(sockfd, (struct sockaddr *)&info, sizeof(info));
+  std::string test_body = "";
+  test_body += "{";
```

Comment:
You can define these constants above. Also, you can use raw string literal, or you can define multiple string and let the compiler concatenate them. See:
https://stackoverflow.com/questions/1135841/c-multiline-string-literal

Reply:
Done